lib/pbench/test/unit/server/test_datasets_inventory.py

```diff
@@ -104,16 +115,16 @@ def test_not_a_file(self, query_get_as, monkeypatch):
     def test_dataset_in_given_path(self, query_get_as, monkeypatch):
         monkeypatch.setattr(FileTree, "find_dataset", self.mock_find_dataset)
         monkeypatch.setattr(Path, "is_file", lambda self: True)
-        monkeypatch.setattr(
-            werkzeug.utils, "send_file", lambda *args, **kwargs: {"status": "OK"}
-        )
+        monkeypatch.setattr(werkzeug.utils, "send_file", self.mock_send_file)
+
         response = query_get_as("fio_2", "1-default/default.csv", HTTPStatus.OK)
         assert response.status_code == HTTPStatus.OK
 
-    def test_get_result_tarball(self, query_get_as, monkeypatch):
+    @pytest.mark.parametrize("key", (None, ""))
```

Comment:
We should probably also add `"/"` to the list.

Reply:
Webb, this little gem was your idea ... `None` suppresses the trailing `/` but `""` gets treated as "a target" and is appended *after* the `/`. So including `"/"` here would result in a double slash, which isn't what we want. 🤣

Reply:
`k = "" if target is None else f"/{target}"`

In this test `(None, "")`, the second parameter does the same thing. It converts the `k="/"` and appends it to the URI (i.e) the final URL will be :

`/datasets/inventory/{dataset_id}/` and this condition will be tested here.

Reply:
I blame Nikhil...I burned all my brain cells up on his review and didn't save enough for this one.  Sorry, Siddardh!  (Thanks you two for keeping me on the road!  🙂)